sklearn/df_clsf.py

```diff
@@ -54,25 +57,49 @@
            'num_trees', 'n_classes', 'accuracy', 'time')
 params.n_classes = len(np.unique(y))
 params.size = size_str(X.shape)
-params.dtype = X.dtype
 
-print_header(columns, params)
-
-# Time fit and predict
 fit_time, _ = time_mean_min(clf.fit, X, y,
                             outer_loops=params.fit_outer_loops,
                             inner_loops=params.fit_inner_loops,
                             goal_outer_loops=params.fit_goal,
                             time_limit=params.fit_time_limit,
                             verbose=params.verbose)
-print_row(columns, params, function='df_clsf.fit', time=fit_time)
-
 predict_time, y_pred = time_mean_min(clf.predict, X,
                                      outer_loops=params.predict_outer_loops,
                                      inner_loops=params.predict_inner_loops,
                                      goal_outer_loops=params.predict_goal,
                                      time_limit=params.predict_time_limit,
                                      verbose=params.verbose)
 acc = 100 * accuracy_score(y_pred, y)
-print_row(columns, params, function='df_clsf.predict', time=predict_time,
+
+if params.output_format == "csv":
+    print_header(columns, params)
+    print_row(columns, params, function='df_clsf.fit', time=fit_time)
+    print_row(columns, params, function='df_clsf.predict', time=predict_time,
           accuracy=acc)
+elif params.output_format == "json":
+    import json
+
+    res = {
+        "lib": "sklearn",
+        "algorithm": "decision_forest_classification",
+        "stage": "training",
+        "data_format": params.data_format,
+        "data_type": str(params.dtype),
+        "data_order": params.data_order,
+        "rows": X.shape[0],
+        "columns": X.shape[1],
+        "classes": params.n_classes,
+        "time[s]": fit_time,
+        "accuracy[%]": acc,
+        "algorithm_paramaters": dict(clf.get_params())
```

Comment:
Should probably change `paramaters` -> `parameters` unless there is any compatibility reason.